packagingenv/Dockerfile.template

```diff
@@ -31,11 +31,7 @@ RUN curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | ba
 # Install prerequisites
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get -y install \
-    build-essential curl gnupg devscripts debhelper dh-make git libicu-dev curl
-
-{%- if version in ('focal') %}
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y install dh-systemd
-{% endif %}
+    build-essential curl gnupg devscripts debhelper dpkg-dev dh-make git libicu-dev curl
```

Comment:
I presume dh-systemd is nolonger required then for focal?

Reply:
Good question.  I just checked, `dh-systemd` is a transition package in focal, which contains nothing but documentation.  `dh_installsystemd` is provided in the `debhelper` package and is used in debian compat levels higher than 10. So we don't need to install `dh-systemd` any longer.  I used [compat level 12](https://manpages.debian.org/testing/debhelper/debhelper-compat-upgrade-checklist.7.en.html#v12) but perhaps 13 could have been used instead but I don't think it'll gain anything for us at this stage.